source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -719,19 +719,21 @@ namespace
                              VectorizedArray<Number> *data)
   {
     Assert(degree > 0, ExcNotImplemented());
-    const unsigned int loop_length = 2*degree+1;
+    const int loop_length = 2*degree+1;
```

Comment:
Even if `loop_length` is `unsigned`, I think the expression `loop_length-1` is signed. You would only have to make sure the loop counter is signed.

But it doesn't matter. Unless you want to make further changes, please merge yourself.